Dockerfile

```
@@ -32,16 +47,22 @@ COPY src/Pipfile src/Pipfile.lock ./
 # VIRTUAL_ENV environment variable if it is set.
 RUN pipenv sync --clear --verbose
 
-FROM python:3.10.7-alpine3.16 AS build-stage
+# The version of Python used here should match the version of the Alpine
+# python3 package installed in the compile-stage.
+FROM python:3.10.10-alpine3.17 AS build-stage
 
 RUN apk --no-cache add \
-  ca-certificates=20220614-r0 \
-  chromium=102.0.5005.173-r0 \
-  libxml2-dev=2.9.14-r1 \
-  libxslt-dev=1.1.35-r0
+  ca-certificates=20220614-r4 \
+  chromium=110.0.5481.177-r0 \
+  libxml2-dev=2.10.3-r1 \
+  libxslt-dev=1.1.37-r0
 
 ENV VIRTUAL_ENV=/task/.venv
+
+# Copy in the Python venv we created in the compile stage and re-symlink
+# python3 in the venv to the Python binary in this image
 COPY --from=compile-stage ${VIRTUAL_ENV} ${VIRTUAL_ENV}/
+RUN ln -sf "$(command -v python3)" "${VIRTUAL_ENV}"/bin/python3
```

Prefer long flag names and alphabetize because why not? 😄 

```suggestion
RUN ln --force --symbolic "$(command -v python3)" "${VIRTUAL_ENV}"/bin/python3
```

---

Since this runs under `ash` the long forms are not available sadly.

---

Doh!  I'll leave it up to you whether or not you change the short flag order- I'm cool either way.